Bio/PDB/parse_pdb_header.py

```diff
@@ -214,7 +214,7 @@ def _parse_pdb_header_list(header):
         # From here, all the keys from the header are being parsed
         if key == "TITLE":
             name = _chop_end_codes(tail).lower()
```

Comment:
Ignoring the fact that we are inheriting a `dict`-named dictionary... I'd make use of the `dict.get` method here:
```
dict_name = dict.get('name', '')  # returns empty string if not found
dict_name = ' '.join([dict_name, name]).strip()  # faster than string concatenation
```

Reply:
> Ignoring the fact that we are inheriting a `dict`-named dictionary... 

As we are in agreement here I have renamed to pdbh_dict

> I'd make use of the `dict.get` method here:
> 
> ```
> dict_name = dict.get('name', '') + ' ' + name
> dict_name = dict_name.strip()
> ```

just pushed a modified version incorporating this - with addition that pdbh_dict{"name"] is defined at the beginning (legacy) to be "" so can just do the (.join()).strip().